### PR TITLE
[WIP] DONT MERGE: Added configuration for Travis

### PR DESCRIPTION
**Pull request by Slash32**

### CODE REVIEW

File: .travis.install-dependencies.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ echo "#!/bin/bash" > ${MAVEN_EXECUTOR}
 echo "set -ev" >> ${MAVEN_EXECUTOR}
 echo "export M2_HOME=/usr/local/apache-maven-${MAVEN_VERSION}" >> ${MAVEN_EXECUTOR}
 echo "export MAVEN_BASEDIR=/usr/local/apache-maven-${MAVEN_VERSION}" >> ${MAVEN_EXECUTOR}
-echo "mvn" "\"\$@\"" >> ${MAVEN_EXECUTOR}
+echo "/usr/bin/mvn" "\"\$@\"" >> ${MAVEN_EXECUTOR}
```
Comment (rasendubi) [inline .travis.install-dependencies.sh:17]: /me cries

**Comment by rasendubi** [comment]
How is this related to KSB-24?

File: .travis.install-dependencies.sh (hunk 1 above)
Comment (Slash32) [reply .travis.install-dependencies.sh:17]: /me too, but have no better idea on how to make it use maven 3.3.9 instead of its 3.2.5 version.

**Comment by Slash32** [comment]
Wrong ticket number and Id. Anyway, the PR is for testing at the moment and will be re-pushed before actual merge.

**Comment by Slash32** [comment]
This became obsolete as #510 replaces this commit.

**Pull request closed without merging**